### ISSUE TITLE
Android 11 latest image and ARM image support

### ISSUE THREAD

**Issue by clgwlg** [google/android-emulator-container-scripts#211]
I am wondering, does the script support Android 11 latest image and ARM image？ If not，Is there a plan to support?

**Comment by 741g** [comment]
Hi, we plan on fixing up the arm64 support in aosp master and in later images, it's possible to also deliver an Android 11 arm64 image but we haven't decided yet.

**Comment by clgwlg** [comment]
Sounds great! Thanks for your reply, Is there a plan to support latest Android 11 x86_64 image?

**Comment by 741g** [comment]
Yes, we plan to re-release Android 11 x86_64 soon. Will get back to you with an ETA.

**Comment by averyzhong** [comment]
Great, looking forward to supporting ARM image.

**Comment by copiltembel** [comment]
> 
> 
> Hi, we plan on fixing up the arm64 support in aosp master and in later images, it's possible to also deliver an Android 11 arm64 image but we haven't decided yet.

Hi, sounds good. Do you have a timeline, at least for the decision on delivering new arm64 images?

**Comment by nahojkap** [comment]
@741g  Hi, would you happen to have an update on this? Thanks!

**Comment by jpcottin** [comment]
Both are supported now:

- Android 11 (R, API 30) appears in `emu-docker list` and is buildable via `emu-docker create`.
- ARM system images surface in the listing when you pass `--arm`.

Closing as addressed.
